src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
Actually it returns a [Promise<Response>](https://github.com/bitinn/node-fetch#fetchurl-options).

However, when specifying that, I get an error when doing:

```
.then(body => body.json())
    .then(info => {
      if (info.loyalty_applicant_at) {
```

with the error `'Property 'loyalty_applicant_at' does not exist on type 'Response'.'`

I'm sort of confused, I guess something is getting confused after I call `json()` on the `Response` (implementing the `Body` interface)? It _does_ work with `any` (or leaving it out altogether).